ui/prometheus-plugin/src/plugins/prometheus-datasource.tsx

```diff
@@ -37,6 +37,7 @@ const createClient: DatasourcePlugin<PrometheusDatasourceSpec, PrometheusClient>
     options: {
       datasourceUrl,
     },
+    healthCheck: healthCheck({ datasourceUrl, headers: specHeaders }),
```

Comment:
For similarity with others requests, I would have add headers as param too. But I guess it's fine 😄 
```suggestion
    healthCheck: (headers) => healthCheck({ datasourceUrl, headers: headers ?? specHeaders }),
```